scripts/nightlyBuild.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
```

Comment:
It seems PROD/RC/nightly build has the similar steps (get aikey, build, package and publish to marketplace), can we share this in one single script? Only the trigger time is different according to pattern of tag.

Reply:
Fixed.